views/sidebar/editprofile-avatar.ejs

```diff
@@ -1,2 +0,0 @@
-<h2>Your profile picture</h2>
-<p>Profile pictures are managed along with your profile on the OpenMRS Wiki. Head over to Confluence to <a href="https://wiki.openmrs.org/users/viewmyprofile.action" target="_blank">edit your Wiki profile</a> or <a href="https://wiki.openmrs.org/users/editmyprofilepicture.action" target="_blank">change your profile picture</a>.</p>
```

Comment:
I've put it [here](https://github.com/SquidDev-GCI/openmrs-contrib-id/commit/0909c826f653b336f7c94994b8f42e66f282c2a1#diff-3d52a10410b37f3c69ef43316662313dR29) instead. The [wireframe](https://wiki.openmrs.org/display/projects/ID+Dashboard+Wireframes#IDDashboardWireframes-Dashboard-EditAccountPage) had it there instead.